hoomd/conftest.py

```diff
@@ -601,6 +601,180 @@ def index_id(i):
     return f"(i={i})"
 
 
+class Options:
+    """Item should be one of a set number of values.
+
+    For use with `Generator`.
+    """
+
+    def __init__(self, *options):
+        self.options = options
+
+
+class Either:
+    """Item should be a value from a set number of specs.
+
+    For use with `Generator`.
+    """
+
+    def __init__(self, *options):
+        self.options = options
+
+
+class Generator:
+    """Class that can generate values of a specified specification.
```

Comment:
Docstring is confusing. I'm not sure what the right wording is.

Reply:
@b-butler please clarify, you wrote this.